### PR TITLE
Add recipe for MultiNEAT

### PR DESCRIPTION
**Pull request by anton-matosov**

### CODE REVIEW

**Comment by anton-matosov** [comment]
Ready to merge

File: recipes/multineat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - setuptools
+    - cmake
+    - psutil
+    - boost=1.66*
+    - boost-cpp=1.66*
+  run:
+    - python
+    - psutil
+    - boost=1.66*
+    - boost-cpp=1.66*
```
Comment (scopatz) [inline recipes/multineat/meta.yaml]: Do you really need a runtime pin here?

File: recipes/multineat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - numpy>=1.11.3
+    - matplotlib=2.2*
+    - opencv=3.3*
```
Comment (scopatz) [inline recipes/multineat/meta.yaml]: Shoudln't these be `>=` conditions?

Comment (anton-matosov) [reply recipes/multineat/meta.yaml]: Fixed

File: recipes/multineat/meta.yaml (hunk 1 above)
Comment (anton-matosov) [reply recipes/multineat/meta.yaml]: Updated to min requirement of >=1.65

Comment (anton-matosov) [reply recipes/multineat/meta.yaml]: Seems to be not working... 1.67 has different naming convention for boost-python. So only 1.65 and 1.66 are compatible at the moment.
Is there a way to specify version range?

**Comment by anton-matosov** [comment]
Hi @scopatz, I have applied all your comments and this PR is ready for the next round of review.

**Comment by scopatz** [comment]
Thanks @anton-matosov!

**Pull request merged**